tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -18,15 +18,15 @@ def test_process_ses_results(sample_email_template):
 
 
```

Comment:
Do we need a test that if the service doesn't have a url the task is not created?